tests/regression/46-apron2/96-witness-mm-escape2.t

```diff
@@ -0,0 +1,22 @@
+  $ goblint --disable ana.dead-code.lines --disable warn.race --enable warn.deterministic --disable warn.behavior --set ana.activated[+] apron --set ana.path_sens[+] threadflag --set ana.relation.privatization mutex-meet-tid-cluster12 --enable witness.yaml.enabled --disable witness.invariant.other --disable witness.invariant.loop-head 96-witness-mm-escape2.c --set witness.yaml.path 96-witness-mm-escape2.yml
+  [Info][Witness] witness generation summary:
+    location invariants: 8
+    loop invariants: 0
+    flow-insensitive invariants: 1
+    total generation entries: 6
+
+  $ goblint --disable ana.dead-code.lines --disable warn.race --enable warn.deterministic --disable warn.behavior --set ana.activated[+] apron --set ana.path_sens[+] threadflag --set ana.relation.privatization mutex-meet-tid-cluster12 --set witness.yaml.validate 96-witness-mm-escape2.yml 96-witness-mm-escape2.c
+  [Warning][Witness] cannot validate entry of type flow_insensitive_invariant
+  [Info][Witness] witness validation summary:
+    confirmed: 8
+    unconfirmed: 0
+    refuted: 0
+    error: 0
+    unchecked: 0
+    unsupported: 1
+    disabled: 0
+    total validation entries: 9
+  [Success][Witness] invariant confirmed: (unsigned long )arg == 0UL (96-witness-mm-escape2.c:8:5)
+  [Success][Witness] invariant confirmed: -128 <= g (96-witness-mm-escape2.c:22:1)
+  [Success][Witness] invariant confirmed: g != 0 (96-witness-mm-escape2.c:22:1)
+  [Success][Witness] invariant confirmed: g <= 127 (96-witness-mm-escape2.c:22:1)
```

Comment:
There's 8 location invariants generated and 8 confirmed, but only 4 messages. What's up with that?
Are there multiple identical invariants for the same location in the witness or something?

Reply:
I don't know. It seems unrelated to the soundness fixes in this PR though. So we could make a new issue if we want to investigate this.